app/scripts/polyfills/array-polyfills.js

```diff
@@ -9,3 +9,13 @@ if (!Array.prototype.find) {
     return undefined;
   };
 }
+
+if (!Array.prototype.includes) {
+  Array.prototype.includes = function (value) {
+    for (var i = 0; i < this.length; i++) {
```

Comment:
Just a minor comment, please write this for loop as follows (length caching):
`for (var i = 0, len = this.length; i < len; i++) {`